tdrs-backend/tdpservice/test/test_admin.py

```diff
@@ -80,3 +82,21 @@ def test_unauth_cant_access_admin(client):
     url = reverse("admin:index")
     response = client.get(url)
     assert response.status_code == status.HTTP_302_FOUND
+
+
+@pytest.mark.django_db
+def test_admin_users_displays_keys(client, admin_user):
+    """Test an authenticated admin_user sees the appropriate content."""
+    client.login(username=admin_user.username, password="test_password")
+    url = reverse("admin:users_user_change", args=(admin_user.id,))
+    response = client.get(url)
+    assert "Staff status" in response.rendered_content
+    assert "Superuser status" in response.rendered_content
+
+
+@pytest.mark.django_db
+def test_superuser_env_var_is_set():
+    """Test to make sure a valid superuser is in the env."""
+    superuser = os.environ.get('DJANGO_SU_NAME')
+    assert superuser is not None
+    assert re.match(r"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)", superuser)
```

Comment:
It might also be useful to assert that the matching user got created:
```
from tdpservice.users.models import User
assert User.objects.filter(username=superuser).exists()
```